docs/cody/clients/install-vscode.mdx

```diff
@@ -207,17 +184,39 @@ When you `@-mention` files to add to Cody’s context window, the file lookup ta
 
-<img
-	src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.8.0/@file-line-numbers.gif"
-	alt="Cody chat interface @'ing a file"
-	style={{ aspectRatio: '782 / 580' }}
-	/>
+Cody defaults to showing @-mention context chips for all the context it intends to use. When you open a new chat, Cody will show context chips for your current repository and current file (or file selection if you have code highlighted).
+
+### Context retrieval
+
+When you start a new Cody chat, the chat input window opens with a default `@-mention` context chips for all the context it intends to use. This context is based on your current repository and current file (or a file selection if you have code highlighted).
+
+![context-retrieval](https://storage.googleapis.com/sourcegraph-assets/Docs/context-retrieval-0624.png)
+
+At any point in time, you can edit these context chips or remove them completely if you do not want to use these as context. Any chat without a context chip will instruct Cody to use no codebase context. However, you can always provide an alternate `@-mention` file or symbols to let Cody use it as a new source of context.
+
+When you have both a repository and files @-mentioned, Cody will search the repository for context while prioritizing the mentioned files.
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto', aspectRatio: '1920 / 1080' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/context-with-chips-0624.mp4" type="video/mp4" />
+</video>
+
+### Rerun prompts with different context
+
+If Cody chat doesn’t provide you with a correct answer on the first attempt, you can re-run a prompt with slight changes to your context. There are buttons that help you re-run and retry prompts with different contexts:
+
+- Public knowledge only: Cody will not use context; it’ll only use knowledge from the base model
+- Current file only: Re-run the prompt again using just the current file as context
+- Add context: Provides @-mention context options to improve the response. Could be files, symbols, remote repositories, or even web URLs
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto', aspectRatio: '1920 / 1080' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/rerun-context-0624.mp4" />
+</video>
 
 ## Context fetching mechanism
 
 VS Code users on the Free or Pro plan can leverage [local context](/cody/core-concepts/context#context-selection) (local keyword search) as the primary context source for Cody chat with access to local embeddings. Remote embeddings will no longer be produced or used as a context source.
```

Comment:
hmm, we should double check this after the release. I'm not sure what happened to embeddings now lol